external/windows64/README.md

```diff
@@ -20,7 +20,7 @@ The following applications are required when rebuilding the core C++ library, an
 For re-building the nupic.core C++ library the following are required;
 
 - [Microsoft Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/visual-studio-2015-downloads-vs) - Community Free edition (or Enterprise edition)
-- [NumPy C++ headers](http://www.lfd.uci.edu/~gohlke/pythonlibs/) - pip install `numpy-1.9.3+vanilla-cp27-none-win_amd64.whl`
+- [NumPy C++ headers](http://www.lfd.uci.edu/~gohlke/pythonlibs/) - pip install `numpy-1.11.2+vanilla-cp27-none-win_amd64.whl`
```

Comment:
@rcrowder I just guessed on this, because I don't see this wheel listed at http://www.lfd.uci.edu/~gohlke/pythonlibs/. The actual value might actually need to be `numpy-1.11.2+mkl-cp27-cp27m-win_amd64.whl` because it actually exists, but wanted your opinion.